src/OnboardingSPA/pages/Steps/Ecommerce/StepAddress/index.js

```diff
@@ -14,6 +14,7 @@ import countries from '../countries.json';
 import currencies from '../currencies.json';
 import { useWPSettings } from '../useWPSettings';
 import { EcommerceStateHandler } from '../../../../components/StateHandlers';
+import StepStateHandler from '../../../../components/StateHandlers/StepStateHandler';
 
```

Comment:
If the Ecommerce -> StepAddress and stylesheet changes are not a part of the scope of this story, can we remove the changes?

Reply:
hey .. this is from Allen's branch ... he had made the changes for the ecommerce steps ... i created my branch from his branch so his changes are also appearing here

Reply:
As he has made changes with respect to animate too, maybe you could cheery pick the commit where Animate component is added, and not have ecommerce step changes